lambda-packages/dns-validated-domain-identity-handler/package.json

```diff
@@ -18,12 +18,13 @@
     "ts-jest": "29.1.1",
     "ts-node": "10.9.2",
     "tslint": "6.1.3",
+    "tsup": "^8.0.1",
     "typescript": "^5.3.3"
   },
   "scripts": {
     "clean": "rm -rf dist",
     "prebuild": "npm run clean",
-    "build": "tsc -p tsconfig.json",
+    "build": "tsup src/index.ts",
```

Comment:
Current CDK configuration packages everything under `dist/` directory to zip archive.
Which means, we just need to have dependencies under `dist/` directory, like following way:
1. move aws-sdk to dep from dev dep
2. define "postbuild" hook or combine with build hook to copy dependencies
  -  e.g.`"postbuild": "cp package*.json dist/ && cd dist && npm ci && cd .."`
3. that's all. simple, right?

Reply:
I can do that, but with `tsup` it's bundled directly into a single file inside `dist/index.js`and it has 944kb, don't think we need the whole aws-sdk.

Reply:
Yes I know that we can get smaller & standalone js bundle, but I don't prefer bundled script unless we really need it because it makes debugging harder (e.g. locations in stacktrace are also minified - we need to bundle, or inline source map and use sourcemap-support like stuffs to restore original locations), and most importantly, we will be using the AWS SDK (v3) bundled with the lambda runtime again in the next release - we have to revert those changes for bundling in the next release